icinga2/service.go

```diff
@@ -148,3 +164,10 @@ func (s *MockClient) UpdateService(service Service) error {
 	s.mutex.Unlock()
 	return nil
 }
+
+func (s *MockClient) UpdateCustomVars(serviceName string, customVars Vars) error {
```

Comment:
Ideally, we should have some infrastructure which allows us to track the custom variables set for each service in the mock client. I'd probably add a new map `serviceVars map[string]Vars` to `MockClient` which maps from `Service.FullName()` to the custom vars for the service.